modules/configuring-a-pipeline-server.adoc

```diff
@@ -68,6 +68,8 @@ The *Configure pipeline server* dialog appears.
 [IMPORTANT]
 ====
 If you specify incorrect data connection settings, you cannot update these settings on the same pipeline server. Therefore, you must delete the pipeline server and configure another one.
+
```

Comment:
Suggest adding 'these' to this line to make it clear that 'you can only import artifacts to the S3...' applies only to the 2 cases you've defined. e.g.

_If you plan to import artifacts that were not generated by tasks in your pipeline, or you plan to use external artifacts that were not generated by any pipeline, you can only import **these** artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field._

Same comment for modules/storing-data-with-data-science-pipelines.adoc.

Reply:
Resolved.